CompilerSource/compiler/compile.cpp

```diff
@@ -638,7 +632,6 @@ wto << "namespace enigma_user {\nstring shader_get_name(int i) {\n switch (i) {\
   for (size_t i = 0; i < mfgfn.length(); i++)
     if (mfgfn[i] == '\\') mfgfn[i] = '/';
   make += string(" OUTPUTNAME=\"") + mfgfn + "\" ";
-  make += "eTCpath=\"" + MAKE_tcpaths + "\"";
```

Comment:
This doesn't seem to be represented anywhere; afaict, you've deleted this logic completely. Was it unused?

Reply:
only the windows ey file had this var and it was empty

Reply:
Probably vestigial from before you created that portable zip... this is the toolchain path, which would have been the location of MinGW. It's blank in other systems because it would always be in the PATH.